apps/schema/static/schema/swagger.yaml

```diff
@@ -2004,6 +2004,54 @@ components:
               type: JSON object
               example: {}
 
+        specialInstructions:
+          properties:
+            special_instructions:
+              title: special_instructions
+              description: General instructions or information about the shipment
+              type: JSON object
```

Comment:
I am not sure this should be a JSON field, I would think this field should be a text field. Please clarify with Daven